ThePensionsRegulator.Frontend/Styles/_tpr-header-menu.scss

```diff
@@ -0,0 +1,339 @@
+@import '_tpr-variables.scss';
+@import 'govuk/base';
+
+.nav-open_overlay {
+    display: none;
+}
+
+.tpr-mobile-menu-toggle {
+    display: flex;
+    justify-content: flex-end;
+    background-color: $tpr-colour-tolopea;
+    padding: 2rem;
+    flex-direction: row;
+    margin-left: 4rem;
+    padding-right: 0.5rem;
+    padding-left: 0.5rem;
+    margin-top: -0.5rem;
+    margin-bottom: -0.5rem;
+
+    &:focus {
+        outline: $govuk-focus-width solid $govuk-focus-colour;
+    }
+}
+
+.closed {
+    display: inline;
+    padding-left: 0.6rem;
+    background: 0 0;
+    border: none;
+    color: $tpr-colour-white;
+    font-size: 1.1rem;
+    font-family: $govuk-font-family;
+}
+
+.open {
+    background-color: $tpr-colour-whisper;
+    align-items: baseline;
+    padding: 1.8rem;
+    padding-right: 0.5rem;
+    padding-left: 1rem;
+}
+
+.open::before {
+    content: "x";
+    font-size: 1.5rem;
+    font-weight: 700;
+    font-family: $govuk-font-family;
+    color: $tpr-colour-royal-blue;
+}
+
+.tpr-mobile-menu__svg {
+    width: 1.5rem;
+    height: 1.5rem;
+    transform: scaleX(-1);
+    filter: FlipH;
+    vertical-align: bottom;
+    display: flex;
+}
+
+.tpr-mobile-menu__svg.hide {
+    display: none;
+}
+
+.tpr-mobile-menu-toggle .si-glyph-fill {
+    fill: $tpr-colour-white;
+}
+
+.tpr-header-menu-nav {
+    display: none;
+}
+
+%mobile-menu-visible {
+    display: flex;
+    font-size: 1rem;
+    position: absolute;
+    z-index: 1000;
+    background-color: $tpr-colour-white;
+    width: 100%;
+    font-family: $govuk-font-family;
+    border-bottom: 3px solid $tpr-colour-white;
+    font-weight: $tpr-font-weight-regular;
+    flex-direction: column;
+
+    .tpr-header-search.mobile {
+        display: inline;
+        flex-direction: row;
+    }
+
+    .tpr-mobile-menu__header-search-container {
+        background: $tpr-colour-whisper;
+        align-content: center;
+        padding-left: 1rem;
+    }
+
+    .tpr-header-search.mobile .tpr-header-search__field {
+        width: 100%
+    }
+
+    .tpr-header-search.mobile .tpr-header-search-autocomplete {
+        width: 90%;
+    }
+
+    .tpr-header-search.mobile .autocomplete__wrapper {
+        position: relative;
+        width: 100%;
+    }
+
+    @media (max-width: 960px) {
+        .tpr-header-menu-nav.active .tpr-header-search.mobile .tpr-header-search__button {
+            margin-left: -37px;
```

Comment:
I wanted to test this because values in px are always worth checking - do they still work when the base font size is changed? Sometimes yes, but not always.

However, in this case I couldn't find a situation in the browser where this style shows up, so I wonder whether it's required?

I also wonder why we've got all these `.tpr-header-search` selectors in the `_tpr-header-menu.scss` file rather than `_tpr-header-search.scss`?